### PR TITLE
redo light/dark import process to have a correct sass syntax

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

The main concern is current master uses an invalid sass syntax to load light or dark files:
https://github.com/glpi-project/glpi/blob/e5f1c91b84977d979722d830072eabbb7c6c4526/css/includes/_base.scss#L48-L52
This loads with phpscss, but is completly broken on other tools (like dart's sass)
To note, i kept `$is_dark` var as it's set as a global css variable and may be used by other parts of core or @cconard96's plugins

I took the opportunity to do a few  small clean or refactor actions.

### CODE REVIEW

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Comment by cconard96** [comment]
Is there a missing file?

SCSS compilation fails.
`./includes/palette_light` not found for import: /var/www/html/glpi/css/palettes/auror.scss on line 33, at column 1.

**Comment by orthagh** [comment]
> Is there a missing file?

present here
https://github.com/glpi-project/glpi/pull/10348/files#diff-a0e66de263d40788c6e68b8c2264751df9200617120a052b34ca764dfba0599d

**Comment by cconard96** [comment]
Nevermind, it was my fault. New development env and I had bad permissions on the css folder. Git didn't complain about it and just didn't add the files when I did the pull.

File: css/includes/_fonts.scss [hunk 1]
```diff
@@ -33,3 +33,8 @@
    font-family: Inter;
    src: url("../fonts/Inter-Variable/Inter-VariableFont.ttf");
 }
+
+$font-family-sans-serif: inter, -apple-system, blinkmacsystemfont, san francisco, segoe ui, roboto, helvetica neue, sans-serif !default;
+
+$ti-font-path: "../css/lib/tabler/icons/iconfont/fonts";
```
Comment (cedric-anne) [inline css/includes/_fonts.scss:39]:
```suggestion
$ti-font-path: "../css/lib/tabler/icons/iconfont/fonts";

```

**Review by cconard96 — APPROVED** [review]
Tested some of my plugins and they seem to work properly between light and dark themes.

**Pull request merged**